filament/src/details/Engine.cpp

```diff
@@ -260,7 +260,8 @@ void FEngine::init() {
 
 #if FILAMENT_ENABLE_FGDBG
     if (!debug.fgdbg) {
-        debug.fgdbg = std::make_unique<fgdbg::DebugServer>();
+        auto server = std::shared_ptr<fgdbg::DebugServer>();
+        debug.fgdbg = std::make_unique<fgdbg::Session>("sample", server);
```

Comment:
`Server` is a shared resource among multiple `Sessions`
so it gets injected in the `Session` constructor